src/components/toast/toast.tsx

```diff
@@ -11,31 +14,45 @@ const ToastContainer = styled.div`
 
 type Props = {
 	message: string;
+	type: 'default' | 'warning' | 'error' | 'success';
+	timer?: number;
 	onDismiss: () => void;
 };
 
-const Toast = ({ message, onDismiss }: Props) => (
-	<div
+const Toast = ({ message, onDismiss, type }: Props) => {
+	const { state: { theme } } = useStore();
+
+	return <div
 		style={{
-			background: 'LemonChiffon',
+			display: 'flex',
+			alignItems: 'center',
+			background: `${theme.button[type]}`,
+			color: `${theme.font.pure}`,
 			cursor: 'pointer',
-			fontSize: 14,
-			margin: 10,
-			padding: 10
-		}}
+			fontSize: `${fontSize[14]}`,
+			margin: `${spacing[10]}`,
+			padding: `${spacing[10]}`,
+			borderRadius: `${pxToRem(4)}`
```

Comment:
I think it's a good point to refactor now: add a `constants.ts` to the `styles`folder and add `const defaultBorderRadius = pxToRem(6)` there and replace all `borderRadius` in our project with it